python/pyspark/testing/utils.py

```diff
@@ -311,11 +342,28 @@ def assertSchemaEqual(actual: StructType, expected: StructType):
 
     Examples
     --------
+    >>> from pyspark.pandas.utils import default_session
+    >>> spark = default_session()
```

Comment:
Why do we need this?

Reply:
Oh, it's dummy line from my local testing. Will remove it. Thanks for catching this out!